std/format/package.d

```diff
@@ -269,119 +269,242 @@ It's also possible to use positional arguments for $(I width), $(I
 precision) and $(I separator) by adding a number and a $(B
 '$(DOLLAR)') after the $(B '*').
 
+$(SECTION4 Types)
+
+The following table contains detailed descriptions for every
+available combination of type and format character. To avoid
+unnecessary repetition, here are some notes that apply to all or
+several entries:
+
+$(UL
+  $(LI If the format character is upper case, the outcome is identical
+       to the lower case version, but uses upper case letters.)
```

Comment:
"If the format character is upper case, the resulting string will be formatted using upper case letters"